test/oxo.js

```diff
@@ -0,0 +1,465 @@
+const Reverter = require('./helpers/reverter');
+const Asserts = require('./helpers/asserts');
+const Oxo = artifacts.require('./Oxo.sol');
+
+contract('Oxo', function(accounts) {
+    const reverter = new Reverter(web3);
+    afterEach('revert', reverter.revert);
+    const ether = 1000000000000000000;
+    const asserts = Asserts(assert);
+    const OWNER = accounts[0];
+    const player1 = accounts[1];
+    const player2 = accounts[2];
+    const BadPerson = accounts[3];
+    const bets = 15*ether;
+    let oxoInst;
+
+    before('setup', () => {
+        return Oxo.deployed()
+            .then(instance => oxoInst = instance)
+            .then(reverter.snapshot);
+    });
+
+
+    //////////////////////// joinGame //////////////////////////////
+
```

Comment:
do you have a test that checks for possibility of playing another one game after first game's ended?

Reply:
function win & test with new game - done;
answer don't know yet;
but I really need to know this & hope your help

Reply:
answer for what?)

Reply:
for this:
Can you imagine a situation when player2 won't have his money? Like if first player will be a contract instead of human?

Reply:
Hm. I deleted this comment, 'cose in your contract this situation irrelevant. Your draw function is private I can't see how to abuse it. But you can write a contract and try to still money from your contract. Maybe I'm missing something